ement-room.el

```diff
@@ -3450,6 +3456,12 @@ For use in `completion-at-point-functions'."
                         collect (ement-room-id room)
                         collect (ement-room-canonical-alias room))))))
 
+;;;;; Reaction
+
+(defun ement-room-reaction--read-char ()
+  "Prompt user for character to be used as reaction."
```

Comment:
I try to write docstrings according to what functions return, so how about something like, "Prompt user for reaction and return it."